pandas/core/arrays/_mixins.py

```diff
@@ -112,6 +112,18 @@ def _validate_scalar(self, value):
     # ------------------------------------------------------------------------
 
     def view(self, dtype: Dtype | None = None) -> ArrayLike:
```

Comment:
Might be able to use `@doc` decorator here: https://github.com/pandas-dev/pandas/blob/ad190575aa75962d2d0eade2de81a5fe5a2e285b/pandas/core/arrays/_mixins.py#L296

Reply:
good idea, go for it